sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -595,8 +594,14 @@ def task_worker(self):
                     probe_identifier = fvp_dict["command"]
 
                     if probe_identifier == "probe":
-                        update_appdb_port_mux_cable_response_table(port_m, asic_index, appl_db)
-                    delete_port_from_y_cable_command_table(port_m, y_cable_command_tbl[asic_index])
```

Comment:
Looks like this was the only place the function was getting called. Should you also delete the function definition now that it is unused?

Reply:
fixed